snippets/wsgi/app.py

```diff
@@ -14,16 +14,18 @@
 
 import newrelic.agent
 from decouple import config
+from raven.contrib.django.raven_compat.middleware.wsgi import Sentry
 from whitenoise.django import DjangoWhiteNoise
 
-
 application = get_wsgi_application()
 application = DjangoWhiteNoise(application)
 
 # Add media files
 if settings.MEDIA_ROOT and settings.MEDIA_URL:
     application.add_files(settings.MEDIA_ROOT, prefix=settings.MEDIA_URL)
 
+application = Sentry(application)
```

Comment:
Is this part necessary? Basket isn't doing this but it seems to be working fine.